app/controllers/results_controller.rb

```diff
@@ -7,6 +7,7 @@ def create
   end
 
   def show
+    @early_eligibility_selection = session_data.fetch("early_eligibility_selection", nil)
     @model = CalculationResult.new(session_data)
     track_completed_journey(@model)
     track_page_view(page: :view_results)
```

Comment:
I think we also need `@early_eligibility_selection = session_data.fetch("early_eligibility_selection", nil)` in the `download` method (line 20) so that the PDFs, can display the text.

Reply:
fixed, thanks